File: VB/JsonDataSourceSnippets/Code.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace JsonDataSourceSnippets
	Public Class Code
		Private Sub GenerateJsonDataSource()
			Dim jsonDataSource As New DevExpress.DataAccess.Json.JsonDataSource()
			Dim sourceUri As New Uri("https://raw.githubusercontent.com/DevExpress-Examples/DataSources/master/JSON/customers.json")
			jsonDataSource.JsonSource = New DevExpress.DataAccess.Json.UriJsonSource(sourceUri)
			jsonDataSource.Fill()
		End Sub
	End Class
End Namespace

File: CS/JsonDataSourceSnippets/Code.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace JsonDataSourceSnippets
{
    public class Code
    {
        void GenerateJsonDataSource()
        {
            DevExpress.DataAccess.Json.JsonDataSource jsonDataSource = new DevExpress.DataAccess.Json.JsonDataSource();
            Uri sourceUri = new Uri("https://raw.githubusercontent.com/DevExpress-Examples/DataSources/master/JSON/customers.json");
            jsonDataSource.JsonSource = new DevExpress.DataAccess.Json.UriJsonSource(sourceUri);
            jsonDataSource.Fill();
        }
    }
}
